sdk/communication/communication-sms/samples-dev/usingAadAuth.ts

```diff
@@ -48,13 +35,30 @@ export async function main() {
     return;
   }
 
-  const client = new SmsClient(endpoint, new DefaultAzureCredential());
-  const sendResults = await client.send({
-    // Phone numbers must be in E.164 format
-    from: "<from-phone-number>",
-    to: ["<to-phone-number-1>"],
+  // get credentials
+  const credential: TokenCredential = isNode
+    ? new DefaultAzureCredential()
+    : new ClientSecretCredential(
+        process.env.AZURE_TENANT_ID,
```

Comment:
we would need to add this to the samples.env then, and add a small comment on what they're used for (e.g. used for AAD samples)